{{cookiecutter.project_slug}}/docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: "3.7"
+services:
+
+  dev:
+    build: 
+      context: .
+      target: dev
+    image: dev_image
+    volumes:
+      - .:/home/root/.
+    command: /bin/bash
```

```suggestion
    command: bash -c "poetry run python"
```
this change starts the pytheon interpreter within the virtual environment of `poetry`.
would that be desired behavior?

---

Depends on the workflow. For the repl development it is, for e.g. jupter it maybe isn't? It does not hinder other workflows so the changes should be fine at the moment